File: src/main/java/com/paragon/api/event/render/entity/EntityHighlightOnHitEvent.java
package com.paragon.api.event.render.entity;

import me.wolfsurge.cerauno.event.CancellableEvent;

import java.awt.*;

public class EntityHighlightOnHitEvent extends CancellableEvent {

    private Color colour;

    public Color getColour() {
        return colour;
    }

    public void setColour(Color newColour) {
        this.colour = newColour;
    }

}

File: src/main/java/com/paragon/api/event/render/entity/EntityHighlightOnHitEvent.kt
package com.paragon.api.event.render.entity

import me.wolfsurge.cerauno.event.CancellableEvent
import java.awt.Color

/**
 * @author Surge
 */
class EntityHighlightOnHitEvent : CancellableEvent() {

    var colour: Color? = null

}